sherpa/ui/utils.py

```diff
@@ -14454,26 +14647,50 @@ def plot(self,
         Draw the data and residuals for the default dataset and then
         overplot those from dataset 2:
 
-        >>> plot("data", "resid", cols=1, color="black")
+        >>> plot("data", "resid", color="black")
         >>> plot("data", 2, "resid", 2, overplot=True, color="black", alpha=0.5)
 
+        Draw the data and residuals for the default dataset and then
+        overplot those from dataset 2:
+
+        >>> plot("data", "resid", ids=[1, 2], color="black", alpha=[1, 0.5])
+
+        Draw the data and residual plots for datasets 1 and 2 with
+        per-plot options:
```

Comment:
```suggestion
        Draw the data and residual plots for datasets 1 and 2 with
        per-plot options. Here, all data will be shown slight transparent
        (the same value of ``alpha=0.5`` is applied to both datasets), but with
        different line styles: The first dataset will be shown with a dashed line
        and the second with a dash-dotted line as specified in the list of linestyle
        values. The second plot with the residuals will be in orange for both
        datasets, since ``opts2`` only contains a single value and not a list of 
        parameters.
```